packages/stable/src/api/records/recordsApi.ts

```diff
@@ -130,4 +133,34 @@ export class RecordsAPI extends BaseResourceAPI<RecordItem> {
       request
     );
   };
+
+  /**
+   * [Aggregate records from a stream](https://developer.cognite.com/api#tag/Records/operation/aggregateRecords)
+   *
+   * Aggregate data for records from a stream.
+   *
+   * ```js
+   * const response = await client.records.aggregate('my_stream', {
+   *   aggregates: {
+   *     total_count: { count: {} },
+   *     by_category: {
+   *       uniqueValues: {
+   *         property: ['mySpace', 'myContainer', 'category'],
+   *         aggregates: { total: { sum: { property: ['mySpace', 'myContainer', 'amount'] } } }
+   *       }
+   *     }
+   *   }
+   * });
+   * ```
+   */
+  public aggregate = async (
+    streamExternalId: string,
+    request: RecordAggregateRequest
+  ): Promise<RecordAggregateResults> => {
+    const path = this.url(`${streamExternalId}/records/aggregate`);
```

Comment:
To be safe, let's do:

```suggestion
    const path = this.url(`${encodeURIComponent(streamExternalId)}/records/aggregate`);
```

Reply:
Good catch! I applied this to all relevant endpoints as who knows what weird stream externalIds users might end up creating.

Addressed in https://github.com/cognitedata/cognite-sdk-js/pull/1351/commits/156312727babd992c8187be87ac50098deb43fc0